### PR TITLE
chore(release): adjust Camunda versions

### PR DESCRIPTION
**Pull request by tmetzke**
* bumps to Camunda 7.16.0 + related updates (Spring Boot, AssertJ, Assert)
* removes Ant instructions from READMEs as we don't provide Ant properties

related to CAM-13916

### CODE REVIEW

**Review by mboskamp — APPROVED** [review]

**Comment by mboskamp** [comment]
Looks good to me. Jenkins fails because the artifacts are not released yet.

**Comment by falko** [comment]
In that case the `build.properties` should be completely removed. See: https://github.com/camunda/camunda-archetypes/search?q=build.properties

**Comment by tmetzke** [comment]
Good point @falko, thanks for the hint, will do 👍

File: pom.xml [hunk 1]
```diff
@@ -27,11 +27,11 @@
   </modules>
 
   <properties>
-    <version.camunda>7.15.0</version.camunda>
+    <version.camunda>7.16.0-alpha5</version.camunda>
```
Comment (mboskamp) [inline pom.xml:30]: Reminder to change that back to 7.16.0
```suggestion
    <version.camunda>7.16.0</version.camunda>
```

**Review by mboskamp — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -27,11 +27,11 @@
   </modules>
 
   <properties>
-    <version.camunda>7.16.0</version.camunda>
+    <version.camunda>7.16.0-alpha5</version.camunda>
     <version.camunda-test-coverage>0.4.0</version.camunda-test-coverage>
     <version.springBoot>2.5.4</version.springBoot>
     <version.assertj-core>3.19.0</version.assertj-core>
-    <version.camunda-bpm-assert>12.0.0</version.camunda-bpm-assert>
+    <version.camunda-bpm-assert>11.0.0-SNAPSHOT</version.camunda-bpm-assert>
```
Comment (tmetzke) [inline pom.xml:34]:
```suggestion
    <version.camunda-bpm-assert>12.0.0</version.camunda-bpm-assert>
```

**Pull request merged**